core/src/scene/sceneLoader.cpp

```diff
@@ -1160,14 +1158,15 @@ Filter SceneLoader::generateNoneFilter(Node _filter, Scene& scene) {
 }
 
 void SceneLoader::parseStyleParams(Node params, Scene& scene, const std::string& prefix,
-                                   std::vector<StyleParam>& out) {
+                                   std::vector<StyleParam>& out, bool unified) {
 
     for (const auto& prop : params) {
+        unified |= prefix == "text";
         std::string key;
```

Comment:
@bcamper Should we have support for empty `text` property for an icon style, in which case it will use the default text properties specified in the style?